code/game/machinery/computer/law.dm

```diff
@@ -28,19 +42,67 @@
 
 
 /obj/machinery/computer/aiupload/attackby(obj/item/O as obj, mob/user as mob, params)
```

Comment:
I know this was here already, but it'd be nice if you could remove these.
```suggestion
/obj/machinery/computer/aiupload/attackby(obj/item/O, mob/user, params)
```

Reply:
no problem!